src/axom/CMakeLists.txt

```
@@ -106,3 +106,17 @@ if(WIN32)
             TARGET_DEFINITIONS "axom_EXPORTS")
     endforeach()
 endif()
+
+
+message(STATUS "AXOM_DEPRECATED_TYPES: " ${AXOM_DEPRECATED_TYPES})
+if (NOT ${AXOM_DEPRECATED_TYPES} MATCHES "^(WARN|ERROR|ALLOW)$")
```

```suggestion
if (NOT "${AXOM_DEPRECATED_TYPES}" MATCHES "^(WARN|ERROR|ALLOW)$")
```

It is not safe to leave string variables unquoted in CMake. They evaluate to nothing and produce syntax errors.

---

Done.  Thanks.